### PR TITLE
Base implementation of HomeScreenView with minor changes to my 'Choices' classes

### PR DESCRIPTION
**Pull request by foreverwoods**
My Updates:
- Have configured the necessary buttons needed for HomeScreen implementation
- Separated buttons into Panels
- Base implementation of the ActionPerformed method (User clicks "Search Button")
- Minor neatness adjustments to my "Choices" classes, does not necessarily concern functionality.

Update to others:
- N/A

In Progress/Need to do:
- Deciding what "tags" to use for CuisineType
- Decide what to "catch" for the exception
- Implement the main method 
- Bridge the connection between HomeScreenView to ChoicesSortedView (New Issue added)
- Finish/clean up sortChoices (Interactor) and its "filter" implementation

Need to further test:
- sortChoices (Interactor), so ChoiceSortedView as well
- HomeScreenView

### CODE REVIEW

File: src/main/java/HomeScreenView.java [hunk 1]
```diff
@@ -1,43 +1,141 @@
...
+
+        // Sort Method Buttons
+        JPanel sortFields = new JPanel();
+        ButtonGroup sortButtons = new ButtonGroup();
+
+        // Initialize Buttons
+        JRadioButton sortPriceButton = new JRadioButton();
+        JRadioButton sortAvgStarsButton = new JRadioButton();
+        JRadioButton sortNameButton = new JRadioButton();
+
```
Comment (prajogt) [inline src/main/java/HomeScreenView.java:90]: Suggestion: add another radio button for order, for instance if they wanted to sort by price low to high vs high to low

File: src/main/java/HomeScreenView.java [hunk 2]
```diff
@@ -1,43 +1,141 @@
...
-    // Sort Method Buttons
-    JRadioButton sortPriceButton = new JRadioButton();
+        // Search Button
+        JButton searchButton = new JButton("Search");
+        searchField.add(searchButton);
+        searchButton.addActionListener(this);
 
-    JRadioButton sortAvgStarsButton = new JRadioButton();
+        // Drop-Down Menus Options
+        Integer[] pricingOptions = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```
Comment (prajogt) [inline src/main/java/HomeScreenView.java:63]: Currently the implemented priceBucket is on a scale of 0-5 but this can be changed

**Review by prajogt — APPROVED** [review]
Looks good so far, some suggestions noted below

File: src/main/java/HomeScreenView.java (hunk 1 above)
Comment (foreverwoods) [reply src/main/java/HomeScreenView.java:90]: Noted. Will look to implement this in the interactor as well.

File: src/main/java/HomeScreenView.java (hunk 2 above)
Comment (foreverwoods) [reply src/main/java/HomeScreenView.java:63]: I'll make this change. 0-5 is what I originally had but I think we flip-flopped between the range in lab last time, but yeah, it's easy to change.

**Review by Allof02 — APPROVED** [review]

**Review by prajogt — APPROVED** [review]
Changes look good

**Review by JarodGratton — APPROVED** [review]

**Pull request merged**